internal/cli/cmd/dac/setup/setup.go

```diff
@@ -181,8 +218,14 @@ func (o *option) Validate() error {
 func (o *option) Execute() error {
 	logrus.Debugf("Starting DaC setup with Perses %s", o.version)
 
-	// Create the destination folder
-	err := os.MkdirAll(depsRootDstPath, os.ModePerm)
+	// Add the DaC output folder to .gitignore, if applicable
+	err := addOutputDirToGitignore()
+	if err != nil {
+		return fmt.Errorf("error adding the 'built' folder to .gitignore: %v", err)
```

Comment:
I think it's not a mandatory step, so we could just add a warning message instead.

```go
if err := addOutputDirToGitignore(); err != nil {
   logrus.WithError(error).Warningf("unable to add the folder %q to .gitignore", OutputFolderName)
}
```